src/utils/helpers.js

```diff
@@ -75,7 +75,7 @@ const helpers = {
    * Render value to integer or decimal
    *
    * @param {number} amount Amount to render
-   * @param {boolean} isInteger If it's an integer or decimal
+   * @param {boolean} [isInteger=false] If it's an integer or decimal
```

Comment:
The linter was complaining about this parameter being mandatory, so I decided to fix its JSDocs.
Since the `isInteger` is treated as a boolean and, in its context, `undefined` is parsed as `false`, I decided putting this on the JSDocs to be didactic.

A reader will understand now that by not passing this parameter, it will be considered as a decimal by default, even though the code does not explicitly converts `undefined` to `false`.

Reply:
I understand it, but the parameter is required in the method, so it would be wrong for any reader to think he could not pass it.

Why would the linter complain about it? What was the message?

If we decide to move with it, I feel we should explicitly make this parameter optional (which I think does not make sense in this PR)